packages/web3torrent/src/components/torrent-info/download-info/DownloadInfo.tsx

```diff
@@ -66,11 +68,14 @@ const DownloadInfo: React.FC<DownloadInfoProps> = ({
         participantType={'payer'}
       />
       {!totalSpent.isZero() && (
-        <section className="totalPayed">
-          <p>
-            Total Spent: <strong className="total-spent">{prettyPrintWei(totalSpent)}</strong>
-          </p>
-        </section>
+        <>
+          <section className="totalPayed">
+            <p>
+              Total Spent: <strong className="total-spent">{prettyPrintWei(totalSpent)}</strong>
+            </p>
+          </section>
+          <ChannelBudget budget={createMockBudget()} />
```

Comment:
If I am uploading & downloading, it looks like I will see the `<ChannelBudget/>` component twice. Is that what we want?

Reply:
Perhaps it would be best to call the component `<SiteBudget/>` ?

I think it is best thought of as an integral part of the setup of web3torrent, and therefore shown as soon as a budget exists (rather than conditional on what is happening in any particular payment channel, or waiting until torrenting has begun).

Reply:
> If I am uploading & downloading, it looks like I will see the <ChannelBudget/> component twice. Is that what we want?

Good catch. I thought peer can only be download or upload for the demo, so it would only show once. Anyway, I moved the component up level.

> Perhaps it would be best to call the component <SiteBudget/> ?
I renamed it to `SiteBudgetTable` so that it doesn't have a name conflict with `SiteBudget` type.

Reply:
👍 

(The plan is to allow non seeding peers to upload and download simultaneously)